README.md

```diff
@@ -131,14 +131,17 @@ optional arguments:
   -h, --help            show this help message and exit
   --test TEST, -t TEST, --test= TEST, -t= TEST
                         Select the Checks you want to perform. Defaults to
-                        'acdd' if unspecified
+                        'acdd' if unspecified. Versions of standards can be
+                        specified via `-t <test_standard>:<version>`. If
+                        `<version>` is omitted, or is "latest", the latest
+                        version of the test standard is used.
   --criteria [{lenient,normal,strict}], -c [{lenient,normal,strict}]
                         Define the criteria for the checks. Either Strict,
                         Normal, or Lenient. Defaults to Normal.
   --verbose, -v         Increase output. May be specified up to three times.
   --skip-checks SKIP_CHECKS, -s SKIP_CHECKS
                         Specifies tests to skip
-  -f {text,html,json}, --format {text,html,json}
+  -f {text,html,json,json_new}, --format {text,html,json,json_new}
```

Comment:
So json_new isn't backwards compatible but it supports multiple datasets. The plan would be to replace the current json format with the json_new in a future release (maybe 4.0?)

I tested and it works well.

Reply:
Yeah, ultimately I didn't want to have surprising behavior for applications that might rely on the old JSON output.